ospd_openvas/daemon.py

```diff
@@ -1365,9 +1374,20 @@ def exec_scan(self, scan_id: str):
         self.main_db.release_database(kbdb)
 
 
+class NotusParser(CliParser):
+    def __init__(self):
+        super().__init__('OSPD - openvas')
+        self.parser.add_argument(
+            '--notus-feed-dir',
+            default="/var/lib/openvas/notus/advisories",
+            help='Directory where notus feed is placed. Default: %(default)s',
+        )
```

Comment:
I think it would be better to put it into an additional module or integrate it into the OSPD parser

Reply:
Can you elaborate your thought? It is very small so an extra module seems a bit hefty and putting it deep into OSPD into an already long seems a bit confusing to me.

Reply:
It feels wrong to me to have the parser in the demon module, but you can leave it as it is and resolve this conversation